src/docs/GetStarted/Developers/npmScriptTutorial/index.mdx

```diff
@@ -164,14 +164,14 @@ et voilà !
 
 **Mozaic provide you with static assets like fonts, icons or logos.**
 
-Depending on your env, statics assets likes icons or fonts may be served from a diferent URL.
+Depending on your env, statics assets like icons or fonts may be served from a diferent URL.
 
-let's create **a live-reloading dev server** as an exemple.
+let's create **a live-reloading dev server** as an example.
 
 ### A) Copy the statics :
 
 Let's make a script that will clean our `dist/statics/` directory and copy the static assets from the Mozaic `nodes_module` to our project.
-that way, we will be sure that we always have the last version of thoses static assets in our project.
+that way, we will be sure that we always have the last version of those static assets in our project.
```

Comment:
It will allow us to be sure that we always have the last version of these static assets in our project.